Assets/Scripts/Game/LevelGradeHandler.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using System.IO;
+using UnityEngine;
+
+public class LevelGradeHandler
+{
+    private string _gradeDirPath = "Assets/Scripts/Game/Grades";
```

Comment:
Will this also work when building?

Reply:
good question, I'll look into it

Reply:
![image](https://github.com/ApericotStudio/Poltergeist/assets/50752585/4560406c-9df7-45d1-8f79-de97f7c880be)
So um, it works. It just creates the folder where it should put it.